************* Module APIServerUtils
APIServerUtils.py:63: [C0325(superfluous-parens), ] Unnecessary parens after 'if' keyword
APIServerUtils.py:65: [C0325(superfluous-parens), ] Unnecessary parens after 'if' keyword
APIServerUtils.py:69: [C0325(superfluous-parens), ] Unnecessary parens after 'if' keyword
APIServerUtils.py:20: [C0103(invalid-name), initaliseLogger] Invalid function name "initaliseLogger"
APIServerUtils.py:20: [C0111(missing-docstring), initaliseLogger] Missing function docstring
APIServerUtils.py:21: [W0601(global-variable-undefined), initaliseLogger] Global variable 'my_logger' undefined at the module level
APIServerUtils.py:21: [C0103(invalid-name), initaliseLogger] Invalid constant name "my_logger"
APIServerUtils.py:25: [C0103(invalid-name), initaliseLogger] Invalid variable name "LOG_FILENAME"
APIServerUtils.py:48: [C0103(invalid-name), rebuildDockerHostsArray] Invalid function name "rebuildDockerHostsArray"
APIServerUtils.py:48: [C0111(missing-docstring), rebuildDockerHostsArray] Missing function docstring
APIServerUtils.py:50: [C0103(invalid-name), rebuildDockerHostsArray] Invalid variable name "dockerHostsArray"
APIServerUtils.py:58: [C0103(invalid-name), rebuildDockerHostsArray] Invalid variable name "hostIp"
APIServerUtils.py:65: [C0121(singleton-comparison), rebuildDockerHostsArray] Comparison to False should be 'not expr' or 'expr is False'
APIServerUtils.py:77: [C0103(invalid-name), initaliseGlobals] Invalid function name "initaliseGlobals"
APIServerUtils.py:77: [C0111(missing-docstring), initaliseGlobals] Missing function docstring
APIServerUtils.py:78: [W0601(global-variable-undefined), initaliseGlobals] Global variable 'homeDomain' undefined at the module level
APIServerUtils.py:78: [W0601(global-variable-undefined), initaliseGlobals] Global variable 'defaultDockerHost' undefined at the module level
APIServerUtils.py:78: [C0103(invalid-name), initaliseGlobals] Invalid constant name "homeDomain"
APIServerUtils.py:78: [C0103(invalid-name), initaliseGlobals] Invalid constant name "defaultDockerHost"
APIServerUtils.py:86: [C0103(invalid-name), initiliseRedisDB] Invalid function name "initiliseRedisDB"
APIServerUtils.py:86: [C0111(missing-docstring), initiliseRedisDB] Missing function docstring
APIServerUtils.py:88: [W0601(global-variable-undefined), initiliseRedisDB] Global variable 'redisdB' undefined at the module level
APIServerUtils.py:88: [C0103(invalid-name), initiliseRedisDB] Invalid constant name "redisdB"
APIServerUtils.py:94: [C0103(invalid-name), getEnvironmentVariable] Invalid function name "getEnvironmentVariable"
APIServerUtils.py:94: [C0103(invalid-name), getEnvironmentVariable] Invalid argument name "inVariable"
APIServerUtils.py:94: [C0111(missing-docstring), getEnvironmentVariable] Missing function docstring
APIServerUtils.py:99: [W0703(broad-except), getEnvironmentVariable] Catching too general exception Exception
APIServerUtils.py:96: [C0103(invalid-name), getEnvironmentVariable] Invalid variable name "envVariable"
APIServerUtils.py:99: [C0103(invalid-name), getEnvironmentVariable] Invalid variable name "e"
APIServerUtils.py:102: [C0103(invalid-name), getEnvironmentVariable] Invalid variable name "tracebackStr"
APIServerUtils.py:103: [C0103(invalid-name), getEnvironmentVariable] Invalid variable name "traceLines"
APIServerUtils.py:108: [C0103(invalid-name), getNewWorkerURL] Invalid function name "getNewWorkerURL"
APIServerUtils.py:115: [C0103(invalid-name), getWorkerURLforVehicle] Invalid function name "getWorkerURLforVehicle"
APIServerUtils.py:115: [W0613(unused-argument), getWorkerURLforVehicle] Unused argument 'vehicle_id'
APIServerUtils.py:122: [C0103(invalid-name), getAllWorkerURLs] Invalid function name "getAllWorkerURLs"
APIServerUtils.py:129: [C0103(invalid-name), applyHeadders] Invalid function name "applyHeadders"
APIServerUtils.py:129: [C0111(missing-docstring), applyHeadders] Missing function docstring
APIServerUtils.py:139: [C0103(invalid-name), latLonAltObj] Invalid function name "latLonAltObj"
APIServerUtils.py:139: [C0103(invalid-name), latLonAltObj] Invalid argument name "inObj"
APIServerUtils.py:139: [C0111(missing-docstring), latLonAltObj] Missing function docstring
APIServerUtils.py:147: [C0103(invalid-name), distanceInMeters] Invalid function name "distanceInMeters"
APIServerUtils.py:147: [C0111(missing-docstring), distanceInMeters] Missing function docstring
APIServerUtils.py:149: [C0103(invalid-name), distanceInMeters] Invalid variable name "R"
APIServerUtils.py:156: [C0103(invalid-name), distanceInMeters] Invalid variable name "a"
APIServerUtils.py:157: [C0103(invalid-name), distanceInMeters] Invalid variable name "c"
APIServerUtils.py:3: [W0611(unused-import), ] Unused connect imported from dronekit
APIServerUtils.py:3: [W0611(unused-import), ] Unused APIException imported from dronekit
APIServerUtils.py:13: [W0611(unused-import), ] Unused import uuid
APIServerUtils.py:15: [W0611(unused-import), ] Unused import docker
APIServerUtils.py:16: [W0611(unused-import), ] Unused Thread imported from threading
APIServerUtils.py:17: [W0611(unused-import), ] Unused import APIServerCommand
APIServerUtils.py:6: [C0411(wrong-import-order), ] standard import "import logging" comes before "from dronekit import connect, APIException"
APIServerUtils.py:8: [C0411(wrong-import-order), ] standard import "import traceback" comes before "from dronekit import connect, APIException"
APIServerUtils.py:9: [C0411(wrong-import-order), ] standard import "import json" comes before "from dronekit import connect, APIException"
APIServerUtils.py:10: [C0411(wrong-import-order), ] standard import "import time" comes before "from dronekit import connect, APIException"
APIServerUtils.py:11: [C0411(wrong-import-order), ] standard import "import math" comes before "from dronekit import connect, APIException"
APIServerUtils.py:12: [C0411(wrong-import-order), ] standard import "import os" comes before "from dronekit import connect, APIException"
APIServerUtils.py:13: [C0411(wrong-import-order), ] standard import "import uuid" comes before "from dronekit import connect, APIException"
APIServerUtils.py:16: [C0411(wrong-import-order), ] standard import "from threading import Thread" comes before "from dronekit import connect, APIException"
